webpack.mix.js

```diff
@@ -1,11 +1,16 @@
 let mix = require('laravel-mix');
+let eslint = require('eslint-webpack-plugin');
 
 mix
-  .js('resources/js/app.js', 'resources/static/js')
-  .setPublicPath('resources/static')
-  .postCss('resources/css/app.css', 'resources/static/css', [
-    require('tailwindcss'),
-  ]);
+    .js('resources/js/app.js', 'resources/static/js')
+    .setPublicPath('resources/static')
+    .postCss('resources/css/app.css', 'resources/static/css', [
+        require('tailwindcss'),
+    ]);
```

Comment:
Maybe add versioning?
https://laravel.com/docs/9.x/mix#versioning-and-cache-busting

Reply:
done